client/src/containers/user-dropdown/component.tsx

```diff
@@ -82,4 +82,4 @@ const UserDropdown: React.FC = () => {
   );
 };
 
-export default UserDropdown;
+export default UserDropdown;
```

Comment:
do you have correctly set EditorConfig plugin in your VisualCode?? I think an empty line is required by eslint.